idl/foxsi-smex-setup-script.pro

```diff
@@ -13,6 +14,11 @@ foxsi_name = 'FOXSI-SMEX'
 foxsi_number_of_modules = 3
 foxsi_launch_date = '2020/06/01'
 
+foxsi_shutters_thickness_mm = [0, 1.0, 1.5]
+foxsi_detector_thickness_mm = 0.500
+; the blankets are assumed to be made of mylar
+foxsi_blanket_thickness_mm = 0.5
+
 ; the shells that are included in a FOXSI optics module
 foxsi_shell_ids = indgen(40)+1
 
```

Comment:
shouldn't there be an END at the end of this script? I can't run it without adding that.